functions.php

```diff
@@ -100,6 +100,18 @@ function getUser($nick, $conn = null, $client = null) {
   return $user;
 }
 
+function deleteUser($nick, $conn = null, $client = null) {
+  if ($conn) {
+    $sql = "delete from users where login = '$nick'; ";
```

Comment:
There are two SQL commands being executed but you only parametrized one of them. The other on line 106/107 still has the same issue as before.